core/src/main/scala/org/genivi/sota/core/WebService.scala

```diff
@@ -120,9 +120,12 @@ class PackagesResource(resolver: ExternalResolverClient, db : Database)
 
   val route = pathPrefix("packages") {
     get {
-      complete {
-        NoContent
-        //Packages.list
+      parameters('regex.?) { (regex) =>
```

Comment:
Ok, with @stevana's help there is now a regex validation in common/.../refined/SprayJsonRefined.scala.
So if there is a request like "/packages?regex=123)", the response will be:
```
HTTP/1.1 400 Bad Request

The query parameter 'regex' was malformed:
Predicate isValidRegex("123)") failed: Unmatched closing ')' near index 2
123)
  ^
```